crates/ruff_python_parser/resources/invalid/statements/invalid_assignment_targets.py

```diff
@@ -27,17 +27,6 @@
 a < b < c = 42
 foo() = 42
 
-# N.B. It looks like the parser can't generate a top-level
```

Comment:
I talked with @BurntSushi about this comment and it seems to be outdated. Our parser allows a standalone f-string to be present at the top-level.

Reply:
I see the definition in the grammar but I don't know how you would make the parser parse a standalone formatted value

```
    expr = BoolOp(boolop op, expr* values)
         | NamedExpr(expr target, expr value)
         | BinOp(expr left, operator op, expr right)
         | UnaryOp(unaryop op, expr operand)
         | Lambda(arguments args, expr body)
         | IfExp(expr test, expr body, expr orelse)
         | Dict(expr* keys, expr* values)
         | Set(expr* elts)
         | ListComp(expr elt, comprehension* generators)
         | SetComp(expr elt, comprehension* generators)
         | DictComp(expr key, expr value, comprehension* generators)
         | GeneratorExp(expr elt, comprehension* generators)
         -- the grammar constrains where yield expressions can occur
         | Await(expr value)
         | Yield(expr? value)
         | YieldFrom(expr value)
         -- need sequences for compare to distinguish between
         -- x < 4 < 3 and (x < 4) < 3
         | Compare(expr left, cmpop* ops, expr* comparators)
         | Call(expr func, expr* args, keyword* keywords)
         | FormattedValue(expr value, int conversion, expr? format_spec) <------------here
         | JoinedStr(expr* values)
         | Constant(constant value, string? kind)
```